Cargo.toml

```diff
@@ -14,3 +14,6 @@ opt-level = 3
 # Uncomment to profile on Linux:
 # debug = true
 # lto = false
+
+[replace]
+"cssparser:0.11.0" = {git = "https://github.com/servo/rust-cssparser", rev = "lol-replace"}
```

Comment:
Yes, this was only to be able to run on try before https://github.com/servo/rust-cssparser/pull/123 landed, which conversely I didn’t land before making sure the Servo PR was ready, in order to avoid a deadlock where further cssparser changes would be blocked.